### PR TITLE
Update travis build

### PR DESCRIPTION
**Pull request by dasanra**
yarn install is executing twice. Not critical

We can update VM linux version so we start using a recent version of some programs (including python for pip and aws)

### CODE REVIEW

**Review by nicosampler — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 if: (branch = master) OR (branch = development) OR (tag IS present)
+dist: bionic
 language: node_js
 node_js: '12'
 before_install:
-  - yarn install --network-concurrency 1
```
Comment (nicosampler) [inline .travis.yml:6]: Not sure about removing this line. If the package.json has 2 dependencies from Github (instead of npm), the installation will fail.

Comment (mikheevm) [reply .travis.yml:6]: Why is it not a problem in safe-react then?

Comment (mikheevm) [reply .travis.yml:6]: I don't think the concurrency is somehow related to where the dependencies are coming from

**Comment by dasanra** [comment]
We are changing the repo structure. This configuration changes are not necessary any more

**Pull request closed without merging**